### PR TITLE
[AppConfig] Update release version

### PR DESCRIPTION
**Pull request by mssfang**

### CODE REVIEW

**Review by samvaity — APPROVED** [review]

**Comment by samvaity** [comment]
Do you need version changes to the readme as well?

File: sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Release History
 
-## 1.2.0-beta.2 (Unreleased)
+## 1.2.0 (2021-07-08)
+### Bug Fixes
```
Comment (g2vinay) [inline sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md:4]: the header should be Bugs Fixed as per new changelog guidelines

File: sdk/appconfiguration/pom.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
         <dependency>
           <groupId>com.azure</groupId>
           <artifactId>azure-data-appconfiguration</artifactId>
-          <version>1.2.0-beta.2</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;current} -->
+          <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;current} -->
```
Comment (g2vinay) [inline sdk/appconfiguration/pom.xml:22]: update README file as well.

**Review by g2vinay — APPROVED** [review]
Approving pre-emptively.
Left some comments.

**Review by g2vinay — APPROVED** [review]

**Review by conniey — APPROVED** [review]

**Comment by mssfang** [comment]
> Do you need version changes to the readme as well?

yes. changed the `dependency` to `current` will keep the README up to date

**Pull request merged**